src/routes/router.ts

```diff
@@ -63,17 +63,19 @@ RootRouter.use("/microscope/dlq", MicroscopeDlqRouter);
 
 // Maintenance mode - needs to be before all other routes
 RootRouter.use(MaintenanceRouter);
+//
+// RootRouter.get("/create-branch-options", cookieSessionMiddleware, maybeJiraSymmetricJwtMiddleware, GithubCreateBranchOptionsGet);
```

Comment:
You want to remove this line or explain in comments why leave it as comments?

Reply:
removing! thank you!